Docs/Configuration.tex

```diff
@@ -1522,33 +1522,33 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
     type.
   \end{itemize}
 
-  To workaround these limitations this quirk applies memory attributes table
-  permissions to memory map passed to Apple kernel and optionally attempts
+  To workaround these limitations, this quirk applies memory attribute table
+  permissions to the memory map passed to the Apple kernel and optionally attempts
   to unify contiguous slots of similar types if the resulting memory map exceeds
   4 KB.
 
-  \emph{Note 1}: Since many firmwares come with incorrect memory protection
-  table this quirk often comes in pair with \texttt{SyncRuntimePermissions}.
+  \emph{Note 1}: Since several types of firmware come with incorrect memory protection
+  tables, this quirk often comes paired with \texttt{SyncRuntimePermissions}.
 
   \emph{Note 2}: The necessity of this quirk is determined by early boot failures.
-  This quirk replaces \texttt{EnableWriteUnprotector} on firmwares supporting
-  memory attributes table (MAT). This quirk is generally unnecessary when using
-  \texttt{OpenDuetPkg}, but may be required to boot macOS 10.6 and earlier for
-  unclear reasons.
+  This quirk replaces \texttt{EnableWriteUnprotector} on firmware supporting
+  Memory Attribute Tables (MAT). This quirk is usually unnecessary when using
+  \texttt{OpenDuetPkg}, but may be required to boot macOS 10.6, and earlier, for
+  reasons that are not clear.
 
 \item
   \texttt{SetupVirtualMap}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Setup virtual memory at \texttt{SetVirtualAddresses}.
 
-  Select firmwares access memory by virtual addresses after \texttt{SetVirtualAddresses}
-  call, which results in early boot crashes. This quirk workarounds the problem by
+  Some firmware access memory by virtual addresses after a \texttt{SetVirtualAddresses}
+  call, resulting in early boot crashes. This quirk workarounds the problem by
   performing early boot identity mapping of assigned virtual addresses to physical
   memory.
 
-  \emph{Note}: The necessity of this quirk is determined by early boot failures. Currently
-  new firmwares with memory protection support (like OVMF) do not support this quirk due to
+  \emph{Note}: The necessity of this quirk is determined by early boot failures. Currently,
+  new firmware with memory protection support (such as OVMF) do not support this quirk. See
```

Comment:
does not I guess

Reply:
I think "do not" as it seems not talking about only one specific new firmware.